src/components/VKButton.tsx

```diff
@@ -0,0 +1,19 @@
+import getVKCode from "../helpers/parseUrl";
+import { REDIRECT_URL, VK_APP, BACKEND_URL } from "../constants";
+
+const VKButton = () => {
+  const handleRedirect = () => {
+    window.location.href = `https://oauth.vk.com/authorize?client_id=${VK_APP.id}&display=popup&redirect_uri=${REDIRECT_URL}&scope=email&response_type=code&v=5.120&state=4194308`;
```

Comment:
Please, use some method from 'react-router-dom' to make redirect (instaed of window.location.href), e.g.

useNavigate or useHistory, etc.

Reply:
@Terapool, if we leaving our application/site - we can't use `react-router-dom`'s methods. There are used only for navigate inside our application (to not reload the page).

Reply:
Ok, I see, it is external link